src/Transport/Config/AzureServiceBusTransportInfrastructure.cs

```diff
@@ -55,6 +55,11 @@ public override string ToTransportAddress(LogicalAddress logicalAddress)
 
         public override TransportReceiveInfrastructure ConfigureReceiveInfrastructure()
         {
+            if (topology is ForwardingTopology t)
```

Comment:
To address this one, see my comment here https://github.com/Particular/NServiceBus.AzureServiceBus/pull/533#issuecomment-300614334